src/apps/developer-apps/styles/developers.css

```
@@ -0,0 +1,134 @@
+  background: rgba(147, 51, 234, 0.5);
+  border-radius: 4px;
+}
+
+.developers-app ::-webkit-scrollbar-thumb:hover {
+  background: rgba(147, 51, 234, 0.7);
+}
+
+/* Form inputs focus ring */
+.developers-app input:focus,
+.developers-app textarea:focus {
+  ring-width: 2px;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)